src/model/RolePermission.ts

```diff
@@ -50,8 +50,6 @@ export const permissions = {
     ],
     iban: [ROLE.EC_DIRECT_ADMIN, ROLE.EC_ADMIN],
     'commission-packages': [
-        ROLE.EC_DIRECT_ADMIN,
```

Comment:
it's ok at this moment. 

In the future we're going to implement this feature also for ECs.